docs/build-tools/sbt.md

```diff
@@ -95,7 +95,7 @@ libraries.
 // One of:
 //   ~/.sbt/0.13/build.sbt
 //   ~/.sbt/1.0/build.sbt
-bloopExportJarClassifiers.in(Global) := Some(Set("sources"))
```

Comment:
This is still the preferred way. The java property is just for the situation where we want to run from command line.